### PR TITLE
adding another userful apps

### PR DESCRIPTION
**Pull request by linkmk**
I just cloned HRMS code block to install another frappe apps like CRM, helpdesk and insights in this script.

### CODE REVIEW

File: README.md
Comment (flexcomng) [inline README.md:1]: Can you revert the repo name in the commit to the main?

**Review by flexcomng — CHANGES_REQUESTED** [review]

**Comment by fthobe** [comment]
@flexcomng that would actually be a cool addition. Can I just make the correct pull request?

**Comment by flexcomng** [comment]
> @flexcomng that would actually be a cool addition. Can I just make the correct pull request?

@fthobe  yes please. I will close this pull request now so you can make the correct pull request.

**Comment by fthobe** [comment]
Have there been any significant changes we need to consider?

**Comment by flexcomng** [comment]
No significant changes.

**Comment by fthobe** [comment]
@flexcomng done in PR #40

**Pull request closed without merging**